### ISSUE TITLE
Update chapter 8

### ISSUE THREAD

**Issue by vuhung16au** [vuhung16au/101DeepLearning#55]
Similary, update chapter 8 Optimization for Training Deep Models

8.1 Gradient Descent Variants �
-> Add one-liner explanation

8.1.2 Batch Gradient Descent
-> use narrative style. don't use bullet points

8.1.3 Stochastic Gradient Descent (SGD)
-> use narrative style. don't use bullet points

8.1.4 Mini-Batch Gradient Descent
-> use narrative style. don't use bullet points

8.2 Momentum-Based Methods
-> Add one-liner explanation

8.2.2 Momentum
-> use narrative style. don't use bullet points

8.2.3 Nesterov Accelerated Gradient (NAG)
-> use narrative style. don't use bullet points

8.3 Adaptive Learning Rate Methods
-> Add one-liner explanation

8.3.5 Learning Rate Schedules
-> "Other useful schedules" -> use narrative style. don't use bullet points

8.4 Second-Order Methods �
-> Add one-liner explanation

8.4.2 Newton’s Method
-> use narrative style. don't use bullet points

8.4.3 Quasi-Newton Methods
-> use narrative style. don't use bullet points

8.5 Optimization Challenges �
-> Add one-liner explanation

8.5.2 Vanishing and Exploding Gradients
-> use narrative style. don't use bullet points

8.5.3 Local Minima and Saddle Points
-> use narrative style. don't use bullet points

8.5.5 Practical Optimization Strategy
-> use narrative style. don't use bullet points
